scripts/qa_crowbarsetup.sh

```diff
@@ -3404,13 +3404,27 @@ function onadmin_proposal
         done
     fi
     local proposal
-    for proposal in nfs_client pacemaker database rabbitmq keystone swift \
-        ceph monasca glance cinder neutron nova `horizon_barclamp` \
-        ceilometer heat manila trove barbican magnum sahara murano \
-        aodh tempest; do
+    for proposal in nfs_client pacemaker database rabbitmq ; do
         deploy_single_proposal $proposal
     done
 
+    # The database and rabbitmq proposals are known to cause crm
+    # resource failures in HA setups, so apply this temporary cleanup
+    # to avoid failing the whole run.
+    # https://bugzilla.suse.com/show_bug.cgi?id=1030822
+    onadmin_cleanup_db_mq_vips
+
+    for proposal in keystone swift ceph monasca glance cinder neutron nova ; do
+        deploy_single_proposal $proposal
```

Comment:
I explained in my commit message:

> The reason this is only applied for proposals that come after
the nova proposal is that the oncontroller function uses the nova
proposal to determine which node is the controller. If keystone, for
example, causes a failure, we won't know until after the nova proposal
is applied, but we will still have a better chance of getting relevant
logs.

I think we should rework the oncontroller function to not depend on the nova barclamp but I wanted to try to keep this pull request small.